### PR TITLE
CDK app

### PR DESCRIPTION
**Pull request by benhavo**

### CODE REVIEW

File: cdk/cdk.context.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "availability-zones:account=651920002880:region=us-east-1": [
```
Comment (chris-allen) [inline cdk/cdk.context.json:2]: This shouldn't be here. You probably just ran `cdk diff` while auth'd with the wrong account. Delete this file and re-run `cdk diff --no-change-set`

**Review by chris-allen — CHANGES_REQUESTED** [review]
One small thing, but otherwise this looks solid.

**Review by KellyInnovation — APPROVED** [review]

File: cdk/cdk.context.json (hunk 1 above)
Comment (benhavo) [reply cdk/cdk.context.json:2]: updated

**Comment by chris-allen** [comment]
@benhavo Looks like this needs rebasing

**Comment by benhavo** [comment]
> @benhavo Looks like this needs rebasing

@chris-allen Comments addressed

**Review by chris-allen — APPROVED** [review]

**Pull request merged**